src/test/resources/templates/mail/activateFreeTrialString.txt

```diff
@@ -0,0 +1 @@
+As part of your company's trial and evaluation of OncoKB, below is your activation link for a 3 month trial of OncoKB:
```

Comment:
approvalEmailString is purely for testing. Didn't really know a better way to test the output and that the html file was being parsed correctly, so I went with this. It does pose a bit of an inconvenience though since changing the html file would require this test to be updated as well, so there are probably better solutions. What do you think?

Reply:
Look into why the test methods are still calling from production directories, should call from test directories.

Reply:
Turns out the test methods were not calling from the production directories, the dropdown email files were just not found and the test was checking for an empty string, which passed. In the changes I have just passed a test string for subject/body and ensured the corresponding email has the same fields. The HTML parsing is actually tested in `MailServiceIT` anyway, so no need to test in the Slack tests.